src/wp-includes/taxonomy.php

```diff
@@ -191,6 +191,25 @@ function create_initial_taxonomies() {
 			'show_in_rest'      => false,
 		)
 	);
+
```

Comment:
Suggestion: Note this new taxonomy in the function's DocBlock like this:

```
 * @since 5.9.0 Added `'wp_template_part_area'` taxonomy.
```

Odd that other additions weren't already noted there. Note to myself to update that function's revisions DocBlock.